proto/build_event_stream.proto

```diff
@@ -827,6 +824,21 @@ message WorkflowConfigured {
 
   // Bazel commands invoked by the workflow action.
   repeated InvocationMetadata invocation = 5;
+
+  // The repo that was pushed in order to trigger the worflow.
+  // Ex: "https://github.com/some-user/acme"
+  string pushed_repo_url = 6;
+
+  string pushed_branch = 7;
+
+  string commit_sha = 8;
+
+  // The repo into which the pushed branch is being merged (for PRs),
+  // or the pushed_repo_url (for push events).
+  // Ex: "https://github.com/acme-inc/acme"
+  string target_repo_url = 9;
+
+  string target_branch = 10;
```

Comment:
nit: comment these or remove the newlines  between fields if the comment applies to multiple